src/extension/module.js

```diff
@@ -2865,6 +2901,9 @@
       // toggle sidekick
       window.hlx.sidekick.toggle();
     }
+    if(window.hlx.sidekick.isEditor()){
+      sampleRUM('sidekick:open', true);
```

Comment:
not sure we need to collect this particular interaction. This is more about how many times they opened the sidekick while on a page. Versus something actually actioned (create document, preview doc, publish doc). Might be overkill to collect this info since I can't think of any use cases where this would matter.  Unless there are other use cases you have in mind here.

Reply:
If you want to measure time from open to preview/publish for a particular document, this can be helpful. Also if you want to measure the number of sessions that ended without a proper publish or preview action.